ray-operator/controllers/ray/rayservice_controller.go

```diff
@@ -238,24 +243,60 @@ func (r *RayServiceReconciler) inconsistentRayServiceStatus(oldStatus rayv1alpha
 		return true
 	}
 
-	if oldStatus.ApplicationStatus.Status != newStatus.ApplicationStatus.Status ||
-		oldStatus.ApplicationStatus.Message != newStatus.ApplicationStatus.Message {
-		r.Log.Info(fmt.Sprintf("inconsistentRayServiceStatus RayService ApplicationStatus changed from %v to %v", oldStatus.ApplicationStatus, newStatus.ApplicationStatus))
-		return true
+	oldAppNames := make([]string, 0, len(oldStatus.Applications))
```

Comment:
Yes, because we convert responses from Ray 2.3 and Ray 2.4 into the same format (`RayServiceStatus`), so `inconsistentRayServiceStatus` can process that without caring about ray version.

Reply:
Great